context/app/static/js/components/detailPage/ContributorsTable/utils.ts

```diff
@@ -47,65 +61,79 @@ export interface Contributor {
   orcid: string;
 }
 
+export interface Contact extends Contributor {
+  isContact: true;
+}
+
 const humanBooleanToBoolean = (humanBoolean: LegacyBoolean | CEDARBoolean): boolean =>
   humanBoolean === 'TRUE' || humanBoolean === 'Yes';
 
-const normalizeLegacyContributor = (contributor: LegacyContributor): Contributor => ({
+export const normalizeContributor = (contributor: ContributorAPIResponse): Contributor => ({
```

Comment:
Let's get this out, but it would be helpful to add tests after the fact for each schema.